README.md

```diff
@@ -1,8 +1,112 @@
+- Ramin Akhbari, ([@rakhbari](https://github.com/rakhbari)), eBay
+- Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
+- Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
+- Mauricio Salatino [@salaboy](https://github.com/salaboy), [Camunda](https://camunda.com) / [LearnK8s](https://learnk8s.io)
+- Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- Tracy Ragan [@tracyragan](https://github.com/tracyragan), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- Erik Sternerson [@erkist](https://github.com/erkist), doWhile
+- Cameron Motevasselani ([@link108](https://github.com/link108)), [Armory](https://www.armory.io/)
+
+## New Members
+
+Membership to this SIG is open to public and self-declared.
+
+New members are advised to:
+
+- Join the [SIG](https://lists.cd.foundation/g/sig-events) and
+[CDF TOC](https://lists.cd.foundation/g/cdf-toc) mailing lists.
+- Join the [#sig-events channel on CDF Slack](https://cdeliveryfdn.slack.com/archives/C0151BTKEJX) and introduce yourself.
+- Go through the [README.md](README.md) document.
+- Regularly join the SIG meetings.
+- Submit a PR to add yourself to the [members list](#members).
+- Here are various ways to get involved:
+  - Share your thoughts by joining the meetings, posting to the mailing list, discussions forum or Slack channel.
+  - Add a topic you would like to discuss to [the agenda](docs/meetings.md) of upcoming meeting.
+  - Create a [new issue](https://github.com/cdfoundation/sig-events/issues) or start a [discussion](https://github.com/cdfoundation/sig-events/discussions) to start gathering feedback and collaborating.
+  - Choose an issue where [help is needed](https://github.com/cdfoundation/sig-events/issues/labels/help%20wanted) and comment on it expressing interest.
+
+## Communication
+
+- Mailing list: [lists.cd.foundation/g/sig-events](https://lists.cd.foundation/g/sig-events).
+- GitHub Discussions: [cdfoundation/sig-events/discussions](https://github.com/cdfoundation/sig-events/discussions).
+- Slack: #sig-events [slack channel](https://cdeliveryfdn.slack.com/archives/C0151BTKEJX) on the CDF slack.
+
+## Governance
+
+SIG Events is a [CDF Special Interest Group](https://github.com/cdfoundation/toc/tree/master/sigs).
+
+The process SIG Events follows can be seen from [here](https://github.com/cdfoundation/toc/blob/master/GROUPS.md#sigs).
+
+Chairs and the TOC Sponsor of the SIG are
+
+* Emil Bäckmark ([@e-backmark-ericsson](https://github.com/e-backmark-ericsson)), Ericsson - Co-Chair
+* Andrea Frittoli ([@afrittoli](https://github.com/afrittoli)), IBM - Co-chair
+* Isaac Mosquera ([@imosquera](https://github.com/imosquera)), Armory - TOC Sponsor
+
+## Meetings
+
+SIG Events meets every even week on Mondays at 16:00UTC. (*See your timezone [here](https://time.is/1600_in_UTC)*).
```

Comment:
In the PR for the creation of the SIG we stated

> The SIG meets on the 1st and 3rd Monday of each month, at 4 PM CET.

Did we still want the 1st and 3rd Monday?

Reply:
Nice catch, I will fix that!